pkg/security/admission/admission.go

```diff
@@ -216,40 +240,51 @@ func (c *constraint) createProvidersFromConstraints(ns string, sccs []*kapi.Secu
 				continue
 			}
 
```

Comment:
> resolving the ns values is only applicable to something that needs ns allocation. It could be closed since the other flags gate resolving individual values if that helps readability.

Yes, close here.  It helps break this apart into congruent chunks with a precondition.